src/Components/creatorPage/OtherWorks.js

```diff
@@ -68,10 +77,7 @@ function CreatedWorksCard({ authorId }) {
   useEffect(FindPosts, []);
```

Comment:
oh nooo, same issue as below with naming the functions after 'Posts', if you can fix it that would be very helpful TT